truffle-config.js

```diff
@@ -39,49 +42,37 @@ module.exports = {
    */
 
   networks: {
-    root: {
-      // host: 'localhost',
-      // port: 9545,
-      // network_id: '53227'
-
-      // provider: () =>
-      //   new HDWalletProvider(
-      //     mnemonic,
-      //     'https://ropsten.infura.io/v3/70645f042c3a409599c60f96f6dd9fbc'
-      //   ),
-      // network_id: 3,
-      // gas: 7000000,
-      // gasPrice: 10000000000, // 10 gwei
-      // skipDryRun: true
-
+    development: {
+      host: 'localhost',
+      port: 9545,
+      network_id: '*', // match any network
+      skipDryRun: true,
+      gas: 7000000
+    },
+    bor: {
+      provider: () =>
+        new HDWalletProvider(
+          MNEMONIC,
+          'http://localhost:8545'
+        ),
+      network_id: '*', // match any network
+      gasPrice: '0'
+    },
+    mumbaiRoot: {
```

Comment:
Need to change the migration scripts in package.json
I kept root/child network structure for simplicity on migration, and tests.

Reply:
To deploy on Mumbai:

```
truffle migrate --network mumbaiRoot --f 2 --to 2
truffle migrate --network mumbaiChild --f 2 --to 2
```